samtranslator/model/exceptions.py

```diff
@@ -37,6 +43,17 @@ def message(self) -> str:
             len(self.causes)
         )
 
+    @property
```

Comment:
No tests? It's hard to understand at a glance what's going on. Or to ensure the behavior doesn't unexpectedly change, since it's depended on by others.

Reply:
Test added